.gitignore

```diff
@@ -6,6 +6,9 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
+# Auto-test folders
+auto-test-*
```

Comment:
@sinapsist should the auto-test directories not be cleaned up as part of the tests? Should we be trying to make git ignore a set of directories that should be deleted? I figure if anything not having this in here will tip people off if something wasn't cleaned up